ocd-2912.sql

```diff
@@ -0,0 +1,90 @@
+DROP TABLE IF EXISTS openchpl.complaint;
```

Comment:
Can you drop complaint first if it is referencing complaint_type and complaint_status_type?